v19.2/install-client-drivers.md

```diff
@@ -145,56 +145,43 @@ For a simple but complete "Hello World" example app, see [Build a Java App with
 
 **Support level:** Full
 
-You can use the [Gradle build tool](https://gradle.org/) to get all dependencies for your application, including Hibernate.
+You can use [Gradle](https://gradle.org/install) or [Maven](https://maven.apache.org/install.html) to get all dependencies for your application, including Hibernate. Only Hibernate versions 5.4.19 and later support the Hibernate CockroachDB dialect.
 
-To install Gradle on Mac:
+If you are using Gradle, add the following to your `dependencies`:
 
-{% include copy-clipboard.html %}
-~~~ shell
-$ brew install gradle
+~~~
+implementation 'org.hibernate:hibernate-core:5.4.19.Final'
+implementation 'org.postgresql:postgresql:42.2.14'
 ~~~
 
-To install Gradle on a Debian-based Linux distribution like Ubuntu:
+If you are using Maven, add the following to your `<dependencies>`:
 
-{% include copy-clipboard.html %}
-~~~ shell
-$ apt-get install gradle
+~~~
+<dependency>
+    <groupId>org.hibernate</groupId>
+    <artifactId>hibernate-core</artifactId>
+    <version>5.4.19.Final</version>
+</dependency>
+<dependency>
+    <groupId>org.postgresql</groupId>
+    <artifactId>postgresql</artifactId>
+</dependency>
 ~~~
 
-To install Gradle on a Red Hat-based Linux distribution like Fedora:
 
-{% include copy-clipboard.html %}
-~~~ shell
-$ dnf install gradle
-~~~
+You will also need to specify the CockroachDB dialect in your [Hibernate configuration file](https://www.tutorialspoint.com/hibernate/hibernate_configuration.htm). Versions of the Hibernate CockroachDB dialect correspond to the version of CockroachDB installed on your machine. For example, `org.hibernate.dialect.CockroachDB201Dialect` corresponds to CockroachDB v20.1, and `org.hibernate.dialect.CockroachDB192Dialect` corresponds to CockroachDB v19.2.
 
-For other ways to install Gradle, see [its official documentation](https://gradle.org/install).
+All dialect versions are forward-compatible (e.g. CockroachDB v20.1 is compatible with `CockroachDB192Dialect`). In the event of a CockroachDB version upgrade, using a previous version of the CockroachDB dialect will not break an application, but, to enable all features available in your version of CockroachDB, we recommend keeping the dialect version in sync with the installed version of CockroachDB.
```

Comment:
@ericharmeling Thoughts on below clarification, based on [this thread](https://cockroachlabs.slack.com/archives/CPD3ANLMB/p1596736923201100)
```
All dialect versions are forward-compatible (e.g. CockroachDB v20.1 is compatible with `CockroachDB192Dialect`) assuming your application is unaffected by any backward-incompatible changes found in [release notes](https://www.cockroachlabs.com/docs/releases)
```